python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -70,6 +57,7 @@ def __init__(self, host, process_manager, wss_key):
         self._host = host
         self._engine = sgtk.platform.current_engine()
         self._wss_key = wss_key
+        self._logger = sgtk.platform.get_logger("api-v2")
```

Comment:
All sounds good. I still have the logging polish ticket lined up for next sprint. I'll address this comment then.

The reason I made this change right now was just because I was looking for an easier way to parse the logs visually and via search.